apisix/upstream.lua

```diff
@@ -140,6 +150,8 @@ local function create_checker(upstream)
     healthcheck_parent.checker_idx =
         core.config_util.add_clean_handler(healthcheck_parent, release_checker)
 
+    upstream.is_creating_checker = nil
```

Comment:
We should also reset it when this function exits with nil

Reply:
Done